src/linear_algebra/prod.lean

```diff
@@ -234,6 +234,19 @@ begin
   rw [←prod_map_comap_prod, submodule.prod_bot],
 end
 
+variables {M₁ N₁ N₂ N₃ : Type*}
+variables [add_comm_monoid M₁] [add_comm_monoid N₁] [add_comm_monoid N₂] [add_comm_monoid N₃]
+variables [module R M₁] [module R N₁] [module R N₂] [module R N₃]
+variables (f₁₂ : M₁ →ₗ[R] M₂) (f₂₃ : M₂ →ₗ[R] M₃)
+variables (g₁₂ : N₁ →ₗ[R] N₂) (g₂₃ : N₂ →ₗ[R] N₃)
+
+@[simp]
+lemma prod_map_one : linear_map.prod_map (1 : M₁ →ₗ[R] M₁) (1 : N₁ →ₗ[R] N₁) = 1 :=
```

Comment:
I think it would make sense to state this with `linear_map.id`, since you use comp below.

If you really want the `1` version, then it's fine to have that one too; but you should add the version that uses `*` instead of `∘ₗ` to match